src/components/WalletDropdown/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { useWeb3React } from '@web3-react/core'
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+
+import DefaultMenu from './DefaultMenu'
+import { LanguageMenu } from './LanguageMenu'
+import { TransactionHistoryMenu } from './TransactionMenu'
+
+const WalletWrapper = styled.div<{ height: string }>`
+  border-radius: 12px;
+  width: 320px;
+  height: ${(props) => props.height};
+  display: flex;
+  flex-direction: column;
+  font-size: 16px;
+  top: 60px;
+  right: 70px;
+  background-color: ${({ theme }) => theme.backgroundSurface};
+  padding-top: 16px;
+  padding-bottom: 16px;
+`
+
+const WalletDropdown = () => {
+  const { account } = useWeb3React()
+  const [menu, setMenu] = useState<'DEFAULT' | 'LANGUAGE' | 'TRANSACTIONS'>('DEFAULT')
+  const isAuthenticated = !!account
+  const height = isAuthenticated ? '324px' : '200px'
```

Comment:
why this specific height? could we just not set a height here and use padding instead?

Reply:
The design team requested the height of the wallet always stay the same when switch to other menus (ie language/transactions).  So, I hard coded it here

Reply:
but why is it larger if authenticated vs not? i think that's a bit brittle because it would need to change if the ui changed at all. generally this is why i don't love fixed heights. it'd be better if you had fixed heights on the children maybe?